src/containers/SendContainer/SendContainer.js

```diff
@@ -100,8 +101,8 @@ const SendContainer = (props: Props) => {
         {...customSelectorProps}
       />
       <ValueSelectorCard
-        selectorModalTitle="Send"
-        maxLabel="Send max"
+        selectorModalTitle={t('transactions.title.valueSelectorModal')}
+        maxLabel={t('button.sendMax')}
```

Comment:
`button.sendMax` is missing in the translations file.

Reply:
Thanks! Added :)